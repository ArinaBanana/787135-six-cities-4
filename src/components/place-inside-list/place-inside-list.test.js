import React from "react";
import renderer from "react-test-renderer";
import PlaceInsideList from "./place-inside-list.jsx";

const items = [`WI-FI`, `Breakfast`];

it(`Should render Place inside list`, () => {
  const tree = renderer.create(<PlaceInsideList items={items} />).toJSON();

  expect(tree).toMatchSnapshot();
});
