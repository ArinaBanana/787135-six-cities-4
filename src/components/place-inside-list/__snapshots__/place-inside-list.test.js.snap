// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Place inside list 1`] = `
<ul
  className="property__inside-list"
>
  <li
    className="property__inside-item"
  >
    WI-FI
  </li>
  <li
    className="property__inside-item"
  >
    Breakfast
  </li>
</ul>
`;
